pyvista/examples/downloads.py

```diff
@@ -3496,7 +3419,7 @@ def download_notch_displacement(load=True):  # pragma: no cover
 def download_louis_louvre(load=True):  # pragma: no cover
     """Download the Louis XIV de France statue at the Louvre, Paris.
 
-    Statue found in the Napoléon Courtyard of Louvre Palace. It is a
+    Statue found in the Napol?on Courtyard of Louvre Palace. It is a
```

Comment:
Some encoding bug made the accent become a question mark here.

Reply:
Thanks.

Reply:
```suggestion
    Statue found in the Napoléon Courtyard of Louvre Palace. It is a
```